components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -367,6 +382,222 @@
   },
 
   /**
+   * Attaches mousedown observers on newly created suggests.
+   *
+   * @param event the event being observed
+   */
+  onSuggestCreated : function (event) {
+    this.suggestionSelected = false;
+    var _this = this;
+    // A mousedown event occurred in the suggest. Don't want onInputBlurred to perform a search on this loss of focus
+    $(this.container).observe("mousedown", function(event) {
+      _this.suggestionSelected = true;
+    });
+  },
+
+  /**
+   * Reset the suggest variable.
+   *
+   * @param event the event being observed
+   */
+  onDataSubmitted : function (event) {
+    // Prevents the onInputBlurred performing searches unnecessarily, as an item was just selected from dropdown.
+    this.suggestionSelected = true;
+    this.lastSubmitted = event.memo.value || this.fld.value.strip();
+  },
+
+  /**
+   * If no items are selected from the suggest dropdown, checks the suggest widget suggestions for exact matches.
+   * If no exact match is found, performs a server-side search for the exact match.
+   *
+   * @param event the event being observed
+   */
+  onInputBlurred : function (event) {
+    if (this.suggestionSelected || this.fld.value.length &lt; this.options.minchars) {
+      this.suggestionSelected = false;
+      return;
+    }
+    // Prepare the query.
+    var query = this.fld.value.strip();
+
+    // Try to find a match among the suggest items.
+    var matchSubmitted = this.submitDropdownMatch(query);
+
+    // If no match found in suggest widget, perform a server-side search for the exact match.
+    // This will most likely never happen.
+    if (!matchSubmitted) {
+      // Our input didn't change. No need to search again.
+      if (query == this.lastSubmitted &amp;&amp; query.length &gt; 1) {
+        return;
+      }
+      this.latestRequest++;
+      var requestId = this.latestRequest;
+      var pointer = this;
+      clearTimeout(this.ajID);
+      this.ajID = setTimeout( function() { pointer.submitAjaxSearchMatch(query, requestId) }, this.options.delay );
```

Comment:
You don't need to use the delay here, since its purpose is to not trigger too many requests while the user is still typing. But here, the input is not focused anymore, we know the user isn't going to be typing anything right away.